optimade/server/models.py

```diff
@@ -53,24 +58,110 @@ class StructureResource(EntryResource):
     attributes: StructureResourceAttributes
 
 
-class OptimadeResponseMeta(BaseModel):
+class OptimadeProvider(BaseModel):
+    """ Stores information on the database provider of the
+    implementation.
+
```

Comment:
I like how the whitespace is allowed here but removed in the json description... very nice!